app/scripts/modules/Tooltip.js

```diff
@@ -28,6 +34,17 @@ export default class Tooltip {
   }
 
```

Comment:
Tu este chyba uprava v destroy()

```javascript
  destroy() {
    this.tooltips.destroyAll();
  }
```

Po vsetkych ostatnych zmenach hlasi destroyAll ako unresolved.

@adammockor Vedel by si prosim pozriet nejaky quick win fix?

Reply:
...opravene :-) ked tak len prosim pozri ci to je za teba tak ok.

Reply:
Ano dobre opravene :)